replica/table.cc

```diff
@@ -1414,7 +1414,11 @@ table::table(schema_ptr schema, config config, db::commitlog* cl, compaction_man
                          keyspace_label(_schema->ks_name()),
                          column_family_label(_schema->cf_name())
                         )
+#ifndef FORCE_X_LOG2_COMPACTION_GROUPS
     , _x_log2_compaction_groups(_config.x_log2_compaction_groups)
+#else
+    , _x_log2_compaction_groups(FORCE_X_LOG2_COMPACTION_GROUPS)
+#endif
```

Comment:
I think it would be better if we just changed the default for the above config item based on the macro.
See existing precendents: https://github.com/scylladb/scylladb/blob/d9269abf5b50c8f2a0c45334f6169691ef06c9a5/db/config.cc#L234-L244